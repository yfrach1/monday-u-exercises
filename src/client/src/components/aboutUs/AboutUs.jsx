import harelImg from "../../assets/images/Harel.JPG";
import styles from "./AboutUs.module.css";
const AboutUs = () => {
  return (
    <div className={styles.contentPlacement}>
      <img style={{ width: "300px", padding: "0 10px" }} src={harelImg} />
      <div>
        <div style={{ fontSize: "40px" }}>
          Hey, my name is Harel, and I'm proud to present to you my Todo app.
        </div>
        <div style={{ fontSize: "20px" }}>
          <br />
          <br />
          <div> No more excuses and delaying things;</div>
          <div>
            with this app, you will be tracking the tasks you have, both pending
            and done.
          </div>

          <div>
            so let's go and try to add one; let's say.. wash the dishes ðŸ˜†ðŸ˜†
          </div>
        </div>
      </div>
    </div>
  );
};

export default AboutUs;
