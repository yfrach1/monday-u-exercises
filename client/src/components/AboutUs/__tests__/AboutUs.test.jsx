import React from "react";
import renderer from "react-test-renderer";
import AboutUs from "../AboutUs";

describe("AboutUs Snapshot ", () => {
  test("renders AboutUs component", () => {
    const tree = renderer.create(<AboutUs />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
