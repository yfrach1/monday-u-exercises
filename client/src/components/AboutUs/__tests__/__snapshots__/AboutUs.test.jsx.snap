// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs Snapshot  renders AboutUs component 1`] = `
<div
  className="contentPlacement"
>
  <img
    src="Harel.JPG"
    style={
      Object {
        "padding": "0 10px",
        "width": "300px",
      }
    }
  />
  <div>
    <div
      style={
        Object {
          "fontSize": "40px",
        }
      }
    >
      Hey, my name is Harel, and I'm proud to present to you my Todo app.
    </div>
    <div
      style={
        Object {
          "fontSize": "20px",
        }
      }
    >
      <br />
      <br />
      <div>
         No more excuses and delaying things;
      </div>
      <div>
        with this app, you will be tracking the tasks you have, both pending and done.
      </div>
      <div>
        so let's go and try to add one; let's say.. wash the dishes ðŸ˜†ðŸ˜†
      </div>
    </div>
  </div>
</div>
`;
